### DISCUSSION TITLE
How to create a youtubeRule

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#63] [Q&A]
https://github.com/buggyj/tiddlyclip/issues/68#issuecomment-907766500

As mentioned in this comment, how to create a youtubeRule

**Comment by buggyj** [comment]
Most of the time I use the 'snip' for embedding youtube media:
![image](https://user-images.githubusercontent.com/2855795/151191323-43981b6c-df16-492c-9bd3-acdf3b4df878.png)

which appears in my tiddlywiki as:

![image](https://user-images.githubusercontent.com/2855795/151191757-23021baa-4d7f-48ce-bdd7-b3302b49139a.png)
